lang/rust/maturin/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=maturin
+PKG_VERSION:=0.14.10
+PKG_RELEASE:=1
+
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/PyO3/maturin/archive/refs/tags/
+PKG_HASH:=8fc9bcdcb7f1535d5e3e8bb500c348ca1bff5a6dce87b0ab7dbc5a49723da28a
+
+PKG_HOST_ONLY:=1
+PKG_BUILD_PARALLEL:=1
+
+PKG_BUILD_DEPENDS:=rust/host
+
+include ../rust-package.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)
+    SECTION:=lang
+    CATEGORY:=Languages
+    SUBMENU:=Rust
+    CATEGORY:=Languages
+    DEPENDS:=$(RUST_ARCH_DEPENDS) @!SMALL_FLASH @!LOW_MEMORY_FOOTPRINT
```

Comment:
Why do you depend on `SMALL_FLASH` and `LOW_MEMORY_FOOTPRINT`?

Reply:
Inherited, but I assume maturin should stay host-only :)